src/main/java/net/earthcomputer/clientcommands/command/arguments/PredicatedRangeArgument.java

```diff
@@ -0,0 +1,82 @@
+package net.earthcomputer.clientcommands.command.arguments;
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.arguments.ArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.Dynamic2CommandExceptionType;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.advancements.critereon.MinMaxBounds;
+import net.minecraft.network.chat.Component;
+import org.jetbrains.annotations.Nullable;
+
+public interface PredicatedRangeArgument<T extends MinMaxBounds<?>> extends ArgumentType<T> {
```

Comment:
I'm unsure what this class does.

Reply:
its like CRangeArgument but it will throw exceptions if the value is outside a designated range

Reply:
which doesn't work for stacks that have a lower count.. and is stupid. removing!